wrappers/core18.go

```diff
@@ -516,15 +517,44 @@ func DeriveSnapdDBusConfig(s *snap.Info) (sessionContent, systemContent map[stri
 	return sessionContent, systemContent, nil
 }
 
+func isReadOnlyFsError(err error) bool {
+	if err == nil {
```

Comment:
This is correct but I'd re-write it slightly as follows:
```
if e, ok := err.(*os.PathError); ok { err = e }
if e, ok := err.(syscall.Errno); ok { return e == syscall.EROFS }
return false
```
This will handle both a naked Errno and a PathError equally.

Reply:
Yes, I like it, thanks for the suggestion, will apply it.